cmd/ragengine/main.go

```diff
@@ -105,8 +99,8 @@ func main() {
 		// LeaderElectionReleaseOnCancel: true,
 	})
 	if err != nil {
-		klog.ErrorS(err, "unable to start manager")
-		exitWithErrorFunc()
```

Comment:
Without this `exitWithErrorFunc`, will this process exit with expected error code?

Reply:
replaced return with os.Exit(1)